Doc/biopdb_faq.tex

```diff
@@ -1047,27 +1047,27 @@ \subsubsection*{How do I calculate residue depth?}
 Then use the \texttt{ResidueDepth} class. This class behaves as a
 dictionary which maps \texttt{Residue} objects to corresponding (residue
 depth, C$\alpha$ depth) tuples. The C$\alpha$ depth is the distance
-of a residue's C$\alpha$ atom to the solvent accessible surface. 
+of a residue's C$\alpha$ atom to the solvent accessible surface.
 
 Example:
 
 \begin{lyxcode}
 model=structure{[}0{]}
 
-rd=ResidueDepth(model,~pdb\_file)
+rd=ResidueDepth(model)
 
 residue\_depth,~ca\_depth=rd{[}some\_residue{]}
 \end{lyxcode}
 You can also get access to the molecular surface itself (via the \texttt{get\_surface}
-function), in the form of a Numeric python array with the surface points.
+function), in the form of a Numpy python array with the surface points.
 
 
 \subsubsection*{How do I calculate Half Sphere Exposure?}
 
 Half Sphere Exposure (HSE) is a new, 2D measure of solvent exposure.
 Basically, it counts the number of C$\alpha$ atoms around a residue
 in the direction of its side chain, and in the opposite direction
-(within a radius of 13 �). Despite its simplicity, it outperforms
+(within a radius of 13 �). Despite its simplicity, it outperforms
```

Comment:
Something weird has happened to the special characters. I've changed this on the master to avoid the problem: 1891973fe3cf5a4ad7e05f6a9ccd8d5d84f48a44